parm/config/config.wavepostsbs

```diff
@@ -17,7 +17,7 @@ export DOIBP_WAV='NO' # Input boundary points
 export DOFLD_WAV='YES' # Field data
 export DOPNT_WAV='YES' # Station data
 export DOGRB_WAV='YES' # Create grib2 files
-export DOGRI_WAV='NO' # Create interpolated grids
+export DOGRI_WAV='YES' # Create interpolated grids
```

Comment:
Should this setting just be based on whether `waveinterpGRD` is empty?

Reply:
Good question.  In theory this is a separate control but in practice perhaps this and other things should be more linked.    Can we revisit this in a future issue?

Reply:
Yes